public/API/API_GetConsoleIDs.php

```diff
@@ -4,20 +4,36 @@
 
 /*
  *  API_GetConsoleIDs - returns mapping of known consoles
+ *    a : active - 1 for only active systems, 0 for all (default: 0)
+ *    g : only game systems - 1 for only game systems, 0 for all system types (Events, Hubs, etc) (default: 0)
  *
  *  array
  *   object    [value]
- *    string    ID                  unique identifier of the console
+ *    int       ID                  unique identifier of the console
  *    string    Name                name of the console
  *    string    IconURL             system icon URL
+ *    bool      Active              boolean value indicating if the console is active in RA
+ *    bool      IsGameSystem        boolean value indicating if is a game system (not Events, Hubs, etc.)
```

Comment:
nit: These lines don't need to respecify that the value is a boolean. The datatype is already listed.

```suggestion
 *    bool      Active              indicates if the system is active in RA
 *    bool      IsGameSystem        indicates if the system is a game system (not Events, Hubs, etc.)
```